### PR TITLE
boveri Exposure node updates

### PR DESCRIPTION
**Pull request by ganjigajanan**

### CODE REVIEW

File: gdcdictionary/schemas/exposure.yaml [hunk 1]
```diff
@@ -229,11 +235,17 @@ properties:
         $ref:
           - "_terms.yaml#/not_reported/common"
 
+  exposure_duration_years:
+    $ref:
+      - "_terms.yaml#/exposure_duration_years/common"
+    type: integer
+
```
Comment (wteouchicago) [inline gdcdictionary/schemas/exposure.yaml:242]: we might be able to add a min of 0 here.

**Comment by wteouchicago** [comment]
looks good according to DICT-94

File: gdcdictionary/schemas/exposure.yaml (hunk 1 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/exposure.yaml:242]: +1

**Review by wwysoc2 — CHANGES_REQUESTED** [review]

File: gdcdictionary/schemas/exposure.yaml (hunk 1 above)
Comment (ganjigajanan) [reply gdcdictionary/schemas/exposure.yaml:243]: added

**Review by wwysoc2 — APPROVED** [review]

**Pull request merged**